internal/controller/mantlebackup_controller.go

```diff
@@ -49,6 +51,13 @@ const (
 	syncModeIncremental = "incremental"
 )
 
+var (
+	//go:embed script/job-export.sh
+	embedJobExportSh string
+	//go:embed script/job-upload.sh
+	embedJobUploadSh string
```

Comment:
```suggestion
	embedJobExportScript string
	//go:embed script/job-upload.sh
	embedJobUploadScript string
```

Just minor nits. Since these strings are not shells(`Sh`) but scripts, `..Script` is better.